util/util.go

```diff
@@ -40,6 +44,21 @@ func ExpandPath(path string) (string, error) {
 	return filepath.Join(usr.HomeDir, path[1:]), nil
 }
 
+func FindAlias(path string) (string, error) {
+	if len(packageAlias) == 0 {
```

Comment:
if the alias file is empty will this always be true, leading to us opening the file everytime instead of memoizing it?

Reply:
Thanks for the note! I just added one memoization function. It should work smoothly now